packages/web-components/tsconfig.build.json

```
@@ -0,0 +1,5 @@
+{
+    "extends": "./tsconfig.json",
+    "include": ["./"],
+    "exclude": ["node_modules", "dist/**", "**/stories", "**/test", "vite.config.ts"]
```

```suggestion
    "exclude": ["node_modules", "dist/**", "vite.config.ts"]
```
stories and test can be removed they are not present in this package